rdr_service/dao/workbench_dao.py

```diff
@@ -539,7 +539,8 @@ def get_workspaces_with_user_detail(self, **kwargs):
                 # if has creator, calculate by creator; if no creator, calculate by owners
                 if creator_user_id == researcher.userSourceId and researcher_has_verified_institution:
                     workspace_has_verified_institution = True
-                elif creator_user_id is None and researcher.userSourceId in owner_user_ids:
+                elif creator_user_id is None and researcher.userSourceId in owner_user_ids and \
+                    researcher_has_verified_institution:
```

Comment:
Can we just remove the var init for `researcher_has_verified_institution` on line 516, since we aren't using it anywhere downstream, and just use `affiliation.isVerified` on this test in the condition.

Reply:
Nevermind I see the reason for the var.